02-intermediate-front-end/events-and-callbacks/readme.md

```diff
@@ -33,77 +76,592 @@ button.addEventListener("click", handleClickEvent)
 button.addEventListener("click", handleClickEvent)
 ```
 
-### Callbacks
+To explain why that only fired once, let's look at a simpler object, like a car:
+
```

Comment:
Look at lines 75 and 76. Including the same line twice doesn't cause it to be fired twice.